src/pages/fi-accounts/Windows/AccountsWindow.js

```diff
@@ -1,72 +1,81 @@
 // ** Custom Imports
-import Window from 'src/components/Shared/Window'
 import CustomTabPanel from 'src/components/Shared/CustomTabPanel'
+import { CustomTabs } from 'src/components/Shared/CustomTabs'
+import { useState } from 'react'
 import AccountsForm from '../forms/AccountsForm'
 import DimensionsForm from '../forms/DimensionsForm'
 import CreditLimitsForm from '../forms/CreditLimitsForm'
 import AccountBalanceForm from '../forms/AccountBalanceForm'
-import { useState } from 'react'
-
 
 const AccountsWindow = ({
-  onClose,
+  height,
+  recordId,
   labels,
   maxAccess,
-  recordId
+  expanded
 }) => {
-  
-  const [activeTab, setActiveTab] = useState(0)
+  const [activeTab , setActiveTab] = useState(0)
+  const [editMode, setEditMode] = useState(recordId)
 
-  const editMode = !!recordId
+  const [store , setStore] = useState({
+    recordId : recordId || null,
+    currencies: null
+  })
 
-  return (
+  const tabs = [
+    { label: labels.Accounts },
+    { label: labels.Dimensions, disabled: !store.recordId },
```

Comment:
remove the Dimensions tab. In the task it's mentioned to add the credit limit tab and account balance tab only.